ThirdParty/Deferred/Deferred/Deferred.swift

```diff
@@ -31,8 +31,13 @@ open class Deferred<T> {
     private func _fill(value: T, assertIfFilled: Bool) {
         let (filledValue, blocks) = protected.withWriteLock { data -> (T, [UponBlock]) in
             if assertIfFilled {
-                precondition(data.protectedValue == nil, "Cannot fill an already-filled Deferred")
-                data.protectedValue = value
+                // Ecosia: Silencing the precondition and replacing it with a warning
```

Comment:
Might make sense to very briefly mention why here, something like `Ecosia: Commenting since the precondition caused a crash when stored Favicons were "too large"` (not sure if that was actually the case, you can phrase it better 🙈 )

Reply:
Will add a better comment 💪 - thanks!